testsuite/parse/ast-parse/region.verona

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Not sure about the file name. Nothing in here talks about regions.

Reply:
I intended for it to have something about allocation and regions, but there doesn't seem to be anything analogous to `new Foo in r`.